kafka/build_glibc_linux_amd64.go

```diff
@@ -1,5 +1,5 @@
-// +build !dynamic
-// +build !musl
+//go:build !dynamic && !musl
```

Comment:
This file should not be changed
I know gofmt changed it automatically, but since this file is autogenerated, it should not be edited. "DO NOT EDIT" is a go convention, I don't know why they don't take care of this issue in gofmt itself. 
Unfortunately, this means this specific file needs to be reverted.